02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  <title>Khadijah's Bio</title>
 </head>
 <body>
-  <h1>Indian Restaurant</h1>
+  <h1><u>Khadijah's Bio</u></h1>
+  <a ref="">I own a company called Best Lead Generation that
```

Comment:
You have an anchor tag here but you are not linking to anything.  Also, if you are going to use an anchor tag, it would be best to keep the reference text short. Here you have the anchor tags over a whole paragraph of text. The tag needs to look like the example below. Just something to keep in mind.

Here is an example:
`<a href="www.example.com">Click Here</a>`
Don't forget the 'h'.